lib/http.js

```diff
@@ -56,9 +60,31 @@ async function main() {
       });
     })
   });
+
+  app.post(`/wallets`, (req, res) => {
+    let accountId = req.query.accountId;
```

Comment:
we should use `req.body` here afaik . 
query is only using the query string. whereas body is suitable for post requests:
http://expressjs.com/en/api.html#req.body